libs/pubsub/pubsub.go

```diff
@@ -167,8 +172,12 @@ func (s *Server) BufferCapacity() int { return cap(s.queue) }
 // If len(capacities) > 0, its first value is used as the queue capacity.
 //
 // Deprecated: Use SubscribeWithArgs. This method will be removed in v0.36.
-func (s *Server) Subscribe(ctx context.Context,
-	clientID string, query Query, capacities ...int) (*Subscription, error) {
+func (s *Server) Subscribe(
```

Comment:
I humbly request we not do this where it isn't already needed 😢 . We do it all over, but I find this style unreadable and it isn't forced on us here.

Reply:
I find the lack of indentation between the arguments and the return values more unreadable than this. I'm happy to have one long line, but what was here before was definitely less readable (for me)

Reply:
👍🏼 No objection to a long line, IMO.